protocol/x/clob/keeper/liquidations.go

```diff
@@ -35,19 +35,21 @@ func (k Keeper) MaybeGetLiquidationOrder(
 	}
 
 	// The subaccount is liquidatable. Get the perpetual position and position size to liquidate.
-	clobPair, positionSizeBig, err := k.GetPerpetualPositionToLiquidate(ctx, subaccountId)
```

Comment:
Unrelated to this PR - I always felt like returning clob pair here is weird, since the function is called `GetPerpetual...`.

I think longer term, we can deprecate clob pair id in the match liquidation proto (it was added before we have the in memory mapping between clob pair and perpetual). so updating here to make our lives easier in the future